flow-server/src/main/java/com/vaadin/flow/component/polymertemplate/IdMapper.java

```diff
@@ -129,10 +131,10 @@ private Element getElement() {
      *            id of element to attach to
      * @param field
      *            field to attach {@code Element} or {@code Component} to
-     * @param beforeComponentInject
+     * @param beforeInject
```

Comment:
Parameter is missing documentation should be filled even for privates if javadoc added.
```suggestion
     * @param beforeInject
     *            a callback invoked before assigning the element/component to
     *            the field
```

Reply:
This is technically out of scope of this PR since no changes except parameter name has been done.

But ok.